projects/driver_display/inc/driver_display_brightness.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+// Module used to control the brightness of the driver display screens.
```

Comment:
This doesn't actually describe the intended usage or scenario. What needs to be initialized? What hardware are you assuming exists? How are you supposed to use this module? What are its side effects?

Reply:
You're still missing a lot of information.